common/src/test/java/com/microsoft/identity/common/BrokerOAuth2TokenCacheTest.java

```diff
@@ -158,6 +160,7 @@ public MsalOAuth2TokenCache getTokenCache(final IPlatformComponents context,
                 EXPIRES_ON,
                 SECRET,
                 CLIENT_ID,
+                APPLICATION_IDENTIFIER,
                 SECRET,
                 MOCK_ID_TOKEN_WITH_CLAIMS,
                 "1",
```

Comment:
We might want to add coverage of the following cases.

This is what I see - please correct me if i'm wrong.

1. populate cache *with* the identifier -> try to read with a specific identifier = covered. (seems like you kind of converted every tests in OAuth2TokenCache test classese in to this)
2. populate cache without the identifier -> try to read with a specific identifier = not covered.
3. populate cache *with* the identifier -> try to read without a specific identifier = not covered. (do we need this? This is applicable to probably only rollout scenario.)

Reply:
Agree.  I'll add some additional coverage.